### ISSUE TITLE
Audio control unwantedly enables monitor on-board speaker

### ISSUE THREAD

**Issue by justurbo** [MonitorControl/MonitorControl#319]
**TL;DR**

MonitorControl audio controls automatically enable on-board monitor speaker. The result being audio played through monitor audio jack output AND the on-board monitor speaker.

**Issue description**

I use my monitor's (DELL U3818DW) audio jack to plug in my headphones. You can only control headphone volume through monitor settings. When headphones are plugged into the monitor, both headphones and the monitor on-board speaker are audible. There's a monitor setting to disable on-board monitor speaker. When it is **'Disabled'**, only headphones play audio. I want to use MonitorControl to control my headphones audio which are plugged into the monitor, however, MonitorControl sets on-board monitor speaker setting to **'Enabled'**, re-enabling the on-board speaker, making MonitorControl unusable for my use case...

**Fix/Feature**
1. Do not enable on-board monitor speaker when audio is adjusted through MonitorControl
2. Add a '_Enable on-board monitor speaker on audio adjust_' checkbox to alter this behaviour

**Comment by waydabber** [comment]
Hi @justurbo,

sorry to say but unfortunatelly it is probably not possible to fix this as there are no DDC commands that control the speaker and headphone audio separately (theoretically this could be done with the `Audio Speaker Pair Select` DDC MCCS command, but this is almost never used so most probably not supported by your display). This problem lies with the firmware of the display itself and probably there is nothing much to do about it. :(

I recommend using a software based solution (like https://rogueamoeba.com/soundsource/) or simply use a cheap USB audio dongle or the Mac's audio jack output if it is feasible.
